romanian/padurea_violet/padurea_violet.md

```diff
@@ -1,2 +1,3 @@
 Au intrat in padurea violet si dupa 2 ore au ajuns intr-o poiana.
-Aici au intalnit un :
\ No newline at end of file
+Aici au intalnit un :
+  - [crocobalaur](/croco.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.